js/client.js

```diff
@@ -1228,12 +1228,15 @@
 				if (this.className === 'closebutton') return; // handled elsewhere
 				if (this.className.indexOf('minilogo') >= 0) return; // handled elsewhere
 				if (!this.href) return; // should never happen
-				if (this.host === 'play.pokemonshowdown.com' || this.host === 'psim.us' || this.host === location.host) {
+				if (this.host === 'play.pokemonshowdown.com' || this.host === 'replay.pokemonshowdown.com' || this.host === 'psim.us' || this.host === location.host) {
 					if (!e.cmdKey && !e.metaKey && !e.ctrlKey) {
 						var target = this.pathname.substr(1);
 						var shortLinks = /^(rooms?suggestions?|suggestions?|adminrequests?|bugs?|bugreports?|rules?|faq|credits?|news|privacy|contact|dex|insecure)$/;
 						if (target === 'appeal' || target === 'appeals') target = 'view-help-request--appeal';
 						if (target === 'report') target = 'view-help-request--report';
+						if (this.host === 'replay.pokemonshowdown.com' && target !== 'search') {
```

Comment:
```suggestion
						if (this.host === 'replay.pokemonshowdown.com' && target && target !== 'search') {
```

It's funny that you thought of `/search` but forgot `/` which is probably way more common.

Reply:
You're not actually handling it correctly, anyway; you should probably set `target` to `.` so it doesn't try anything weird.